src/app/app.component.ts

```diff
@@ -3,13 +3,23 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
-export class AppComponent {
-  title = 'ShareLink';
+export class AppComponent implements OnInit {
+  readonly DEFAULT_BANNER_DESCRIPTION: string = "Safer, faster, and more stable"
+  bannerDescription = this.DEFAULT_BANNER_DESCRIPTION
+
+  ngOnInit(): void {
+    // Parse parameters as url and move
+  }
+
+  onBannerClick() {
```

Comment:
onBannerClicked() might be preferred? (But I'm not sure)

Reply:
agreen with that